LLMonFHIR/FHIRInterpretationModule.swift

```diff
@@ -96,10 +97,28 @@ class FHIRInterpretationModule: Module, DefaultInitializable, EnvironmentAccessi
     }
     
     
+    /// Updates the schema used by the interpretation module.
+    ///
+    /// By default, this function will delay the actual schema updating, in order to be able to coalesce multiple calls into just one update.
+    ///
+    /// - parameter forceImmediateUpdate: Set this to `true` to disable the delay and instead update the schema immediately.
     @MainActor
-    func updateSchemas() async {
-        await resourceSummary.changeLLMSchema(to: singleResourceLLMSchema)
-        await resourceInterpreter.changeLLMSchema(to: singleResourceLLMSchema)
-        multipleResourceInterpreter.changeLLMSchema(to: multipleResourceInterpreterOpenAISchema)
+    func updateSchemas(forceImmediateUpdate: Bool = false) async {
```

Comment:
We should maybe to a force update before we launch a study or move into the multiple chats view, just to be safe here that it always uses the right schema on launching the chat for a user.